### PR TITLE
fixes #428: Adding hover effect on the buttons

### LINKED ISSUE #428: ✨ Enhancement: Adding hover effect on the buttons.
### Is your feature request related to a problem? Please describe

Adding a hover effect on the buttons of the website make it more visually applying and design-friendly 

### Describe the solution you'd like

Adding hover effect on the buttons with the help of `hover:` tailwind utility classes  

### Describe alternatives you've considered

n/a

### Additional context

n/a

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by VivekJaiswal18**
**What kind of change does this PR introduce?**

feature fix

**Issue Number:**
-  Closes #428 

**Summary**

Added hover effect to the buttons 
 Issue link: #428
**Does this PR introduce a breaking change?**

 No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
     "next-plausible": "^3.11.1",
     "next-sitemap": "^4.2.3",
     "node-ical": "0.16.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
```
Comment (benjagm) [inline package.json:36]: Why this change?

**Review by DhairyaMajmudar — COMMENTED** [review]
It will be a good enhancement if you reduce the duration time to 3ms with the help of `duration-300`

**Comment by DhairyaMajmudar** [comment]
I checked the changes in responsive mode using Firefox and chrome dev tools. I found that its working fine if touch simulation mode is turned off.

File: pages/index.page.tsx [hunk 2]
```diff
@@ -417,7 +417,7 @@ const Home = (props: any) => {
               <div>
                 <Link
                   href={`/blog/posts/${blogPosts[0].slug}`}
-                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
+                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
```
Comment (DhairyaMajmudar) [inline pages/index.page.tsx:420]:
```suggestion
                  className=' w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
```

File: pages/index.page.tsx [hunk 3]
```diff
@@ -472,7 +472,7 @@ const Home = (props: any) => {
                 </div>
                 <a
                   href='https://calendar.google.com/calendar/u/0/embed?src=c_8r4g9r3etmrmt83fm2gljbatos@group.calendar.google.com'
-                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center pt-1 flex items-center justify-center'
+                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 flex items-center justify-center'
```
Comment (DhairyaMajmudar) [inline pages/index.page.tsx:475]:
```suggestion
                  className='w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 flex items-center justify-center'
```

File: pages/index.page.tsx (hunk 2 above)
Comment (DhairyaMajmudar) [reply pages/index.page.tsx:420]: suggesting these changes to remove vs code warnings coming from using same tailwind utility classes

**Comment by benjagm** [comment]
@VivekJaiswal18 Can you please take a look to the feedback provided in the review? Before merge you'll need to fix the merge conflicts as well.

File: package.json (hunk 1 above)
Comment (benjagm) [reply package.json:36]: @VivekJaiswal18 Why this change?

**Comment by DhairyaMajmudar** [comment]
@VivekJaiswal18 if you need any help feel free to ask : )

**Comment by VivekJaiswal18** [comment]
@benjagm 

> @VivekJaiswal18 Can you please take a look to the feedback provided in the review? Before merge you'll need to fix the merge conflicts as well.

Okay @benjagm looking into it..sorry for the delay!

File: package.json (hunk 1 above)
Comment (VivekJaiswal18) [reply package.json:36]: @benjagm  I introduced this change so that if minor patches were introduced by the library then anyone installing it can benefit from it (if anyone does npm install react/react-dom). 
But if you think it should not be done then I can remove it.

File: pages/index.page.tsx (hunk 2 above)
Comment (VivekJaiswal18) [reply pages/index.page.tsx:420]: I don't think it will give any errors due to - using same tailwind utility classes but anyway it is obsolete so should be removed.

File: package.json (hunk 1 above)
Comment (DhairyaMajmudar) [reply package.json:36]: I think its good to remove that change since it also modifies files like `package-lock.json` and `yarn.lock` and ultimately increases the PR size.

image

**Comment by VivekJaiswal18** [comment]
PR for this issue is provided in #497 considering the Feedback provided in the review
